marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/WriteStepRunner.java

```diff
@@ -252,10 +254,9 @@ public Job run() {
             e.printStackTrace();
             job.setCounts(0,0, 0, 0, 0)
                 .withStatus(JobStatus.FAILED.toString());
-            StringWriter errors = new StringWriter();
-            e.printStackTrace(new PrintWriter(errors));
-            job.withStepOutput(errors.toString());
-            //TODO: create job doc if not present
+            //StringWriter errors = new StringWriter();
+            //e.printStackTrace(new PrintWriter(errors));
+            job.withStepOutput(e.getLocalizedMessage());
```

Comment:
We want to write the exception stack trace so that users can identify the issue not just the message

Reply:
well, we can discuss this Monday...